OpenRA.Game/Graphics/IGraphicsDevice.cs

```diff
@@ -30,6 +30,7 @@ public RendererAttribute(Type graphicsDeviceType)
 	public interface IDeviceFactory
 	{
 		IGraphicsDevice Create(Size size, WindowMode windowMode);
+		ISoundEngine Initialize();
```

Comment:
Can we name these consistently?  `CreateGraphics` / `CreateSound` or `InitializeGraphics` / `InitializeSound`.